pipelines/release-to-github-signed/release-to-github-signed.yaml

```diff
@@ -0,0 +1,359 @@
+            CAN_I_READ_SNAPSHOTS=$(kubectl auth can-i get snapshot/${SNAPSHOT_NAME} -n ${ORIGIN_NAMESPACE})
+            CAN_I_CREATE_INTERNALREQUESTS=$(kubectl auth can-i create internalrequest -n ${TARGET_NAMESPACE})
+
+            echo ""
+            echo "CAN_I_READ_RELEASES? ${CAN_I_READ_RELEASES}"
+            echo "CAN_I_READ_RELEASEPLANS? ${CAN_I_READ_RELEASEPLANS}"
+            echo "CAN_I_READ_RELEASEPLANADMISSIONS? ${CAN_I_READ_RELEASEPLANADMISSIONS}"
+            echo "CAN_I_READ_SNAPSHOTS? ${CAN_I_READ_SNAPSHOTS}"
+            echo ""
+            echo "CAN_I_CREATE_INTERNALREQUESTS? ${CAN_I_CREATE_INTERNALREQUESTS}"
+            echo ""
+
+            if [ "${CAN_I_READ_RELEASES}" == "no" ] ||\
+                [ "${CAN_I_READ_RELEASEPLANS}" == "no" ] ||\
+                [ "${CAN_I_READ_RELEASEPLANADMISSIONS}" == "no" ] ||\
+                [ "${CAN_I_READ_SNAPSHOTS}" == "no" ] ||\
+                [ "${CAN_I_CREATE_INTERNALREQUESTS}" == "no" ] ; then
+              echo "Error: Cannot read or create required Release resources!"
+              echo ""
+              echo "This indicates that your workspace is not correctly setup"
+              echo "Please reach out to a workspace administrator"
+              exit 1
+            fi
+
+            echo "Access to Release resources verified"
+    - name: collect-data
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-catalog.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/collect-data/collect-data.yaml
+      params:
+        - name: release
+          value: $(params.release)
+        - name: releaseplan
+          value: $(params.releaseplan)
+        - name: releaseplanadmission
+          value: $(params.releaseplanadmission)
+        - name: snapshot
+          value: $(params.snapshot)
+        - name: subdirectory
+          value: $(context.pipelineRun.uid)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - verify-access-to-resources
+    - name: validate-single-component
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-catalog.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/validate-single-component/validate-single-component.yaml
+      params:
+        - name: snapshotPath
+          value: "$(context.pipelineRun.uid)/snapshot_spec.json"
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-data
+    # - name: verify-enterprise-contract
+    #   taskRef:
+    #     resolver: "bundles"
+    #     params:
+    #       - name: bundle
+    #         value: $(params.verify_ec_task_bundle)
```

Comment:
I see you have the right parameter here, which is good (you can resolve this). This is the one that should be in the README